spec/extensions/linear_algebra_functions.md

```diff
@@ -77,7 +77,7 @@ Accordingly, the standardization process affords the opportunity to reduce inter
 (function-linalg-cholesky)=
 ### linalg.cholesky(x, /, *, upper=False)
 
-Returns the Cholesky decomposition of a symmetric positive-definite matrix (or a stack of symmetric positive-definite matrices) `x`.
+Returns the lower (resp. upper) Cholesky decomposition `x = LLᵀ` (resp. `x = UᵀU`) of a symmetric positive-definite matrix (or a stack of matrices) `x`, where `L` is a lower-triangular matrix or a stack of matrices (resp. `U` is an upper-triangular matrix or a stack of matrices).
```

Comment:
If you're going to use Unicode symbols I think it would be better to avoid wrapping it in a code block. At least for me, whatever font this uses in the rendered version doesn't look very good. And anyway, backticks should only be used for things that are actual code.

Reply:
I completely agree. I did not know how to handle this one.

It is true that backticks are often used for, code, but I read that the expression `x = USV` was in the SVD docs. If we wanted to go for actual code, that'd have to be te much more convoluted `x = U.matmul(S.matmul(V))`.

Should I just remove the backticks in all these expressions, or what would be the preferred solution?

Reply:
Instead of code blocks, you could make equations italic. In general, would be nice to have equation rendering capabilities in the spec.

Reply:
The problem with making it italics is that `x`, `L` and `U` are represented later in code, so it'd be a bit odd to have some parts of the text in italics and some others in code.

Having LaTeX support would certainly be helpful. The way we did it in PyTorch is to first write the equations in LaTeX and then saying "the outputs map to the equations above as follows". See for example the return section of [torch.linalg.eig](https://pytorch.org/docs/stable/generated/torch.linalg.eig.html)

Reply:
```suggestion
Returns the lower (upper) Cholesky decomposition x = LLᵀ (x = UᵀU) of a symmetric positive-definite matrix (or a stack of matrices) `x`, where `L` is a lower-triangular matrix or a stack of matrices (`U` is an upper-triangular matrix or a stack of matrices).
```